runner/index.js

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env node
 import fs from 'node:fs';
+import util from 'node:util';
```

Comment:
I would prefer to just `console.log`, wdyt?

Reply:
I just copied what the REPL does, but if you'd like I can change that

Reply:
Oh lol sorry, no clue why it does that actually, I'll change repl too later